erigon-lib/state/version_schema.go

```diff
@@ -0,0 +1,71 @@
+package state
+
+import (
+	"github.com/erigontech/erigon-lib/downloader/snaptype"
+)
+
+func InitSchemas() {
+	Schema.AccountsDomain.version.DataKV = snaptype.V1_0
```

Comment:
possible way to simplify this:
step1. loop over all domains; and set them to v1_0
step2. set the non-default values for `Schema.ReceiptDomain.hist.iiCfg.version.DataEF` etc.